### PR TITLE
propagate create_slack_integration setting to library permissions

### PR DESCRIPTION
**Pull request by camhashemi**
@ryanpbrewster @leogrim

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PermissionCommander.scala [hunk 1]
```diff
@@ -119,7 +119,9 @@ class PermissionCommanderImpl @Inject() (
   def combineOrganizationAndLibraryPermissions(lib: Library, libPermissions: Set[LibraryPermission], orgPermissions: Set[OrganizationPermission]): Set[LibraryPermission] = {
     val addedPermissions = Map[Boolean, Set[LibraryPermission]](
       (lib.canBeModified && libPermissions.contains(LibraryPermission.VIEW_LIBRARY) && orgPermissions.contains(OrganizationPermission.FORCE_EDIT_LIBRARIES)) ->
-        Set(LibraryPermission.EDIT_LIBRARY, LibraryPermission.MOVE_LIBRARY)
+        Set(LibraryPermission.EDIT_LIBRARY, LibraryPermission.MOVE_LIBRARY, LibraryPermission.DELETE_LIBRARY),
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PermissionCommander.scala:122]: Are we sure that they can actually delete the library?

**Comment by ryanpbrewster** [comment]
lgtm

Nice test, too. Good call.

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PermissionCommander.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PermissionCommander.scala:122]: asked @JRuff42 , she says the `edit library` feature lets them to whatever an owner can

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PermissionCommander.scala:122]: `LibraryCommander.deleteLibrary` actually checks `lib.ownerId == requesterId`, so we should either change that or change this.

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PermissionCommander.scala:122]: I'll change it there. Trying to keep our logic to spec to reduce confusion between us and product.

**Pull request merged**